services/githubService.js

```diff
@@ -281,7 +281,7 @@ const fetchIssuesById = async (repositoryName, issueId) => {
       logger.error(`GitHub API request failed. Status: ${res.status}, URL: ${url}`);
       return null;
     }
-    return res.json();
+    return await res.json();
```

Comment:
nit : return can be on next line.  It's unusual to see return and await on the same line